client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -218,50 +228,132 @@ h2.action-panel__title {
 			}
 		}
 	}
-
-	.email-provider-features {
-		margin-bottom: 2.5em;
-
-		@include breakpoint-deprecated( '>1040px' ) {
-			margin-bottom: 14px;
-			margin-left: 64px;
-			margin-top: 0;
-		}
-	}
 }
 
 .email-provider-stacked-card__provider-right-panel {
-	width: $width-right-panel-card;
+	margin-left: 24px;
 
 	img {
 		margin-top: 40px;
 	}
 }
 
-.action-panel__body {
-	margin-left: 24px;
-}
-
 .email-provider-stacked-card__title {
 	margin-top: 4px;
 	font-size: 1.25rem;
 	line-height: 24px;
 	color: var( --color-neutral-70 );
 }
 
-.email-provider-stacked-card__header {
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	justify-content: space-between;
+.email-provider-stacked-card__title-container {
+	p {
+		margin-bottom: 0;
+		margin-right: 10px;
+	}
+}
+
+.email-providers-stacked-comparison__provider-card {
+	.email-provider-stacked-card__header {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+
+		@include break-small {
+			flex-direction: row;
+		}
+
+		.email-provider-stacked-card__provider-card-main-details {
```

Comment:
Nit: there is another selector for `.email-provider-stacked-card__provider-card-main-details` on line 125 -- I think it will be much easier to understand what is going on by consolidating the two of them or at least co-locating them in the file.